### DISCUSSION TITLE
internal/modules/cjs/loader:1078

### DISCUSSION THREAD

**Discussion by swarnimbandekar** [PrismarineJS/mineflayer#3073] [Q&A] [answered]
It is saying that the module is not installed and this error is occuring, please do help me asap

![error](https://github.com/PrismarineJS/mineflayer/assets/68631900/6a9b3571-6717-438e-9602-5c4dce898adb)

**Comment by rom1504** [comment]
Use npm install

**Comment by SonicandTailsCD** [comment, marked as answer]
> It is saying that the module is not installed and this error is occuring, please do help me asap
> 
> ![error](https://user-images.githubusercontent.com/68631900/244875238-6a9b3571-6717-438e-9602-5c4dce898adb.png)

You need to install the "minecraft-colors" module. Run this in a terminal:
```cmd
npm install minecraft-colors
```
Hope it helps! Let me know if you have issues. :)
